digdag-tests/src/test/java/acceptance/GroupRetryIT.java

```diff
@@ -0,0 +1,208 @@
+package acceptance;
+
+import com.google.common.io.Resources;
+import io.digdag.client.DigdagClient;
+import io.digdag.client.api.Id;
+import io.digdag.client.api.RestSessionAttempt;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import utils.CommandStatus;
+import utils.TemporaryDigdagServer;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Arrays.asList;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static utils.TestUtils.getAttemptId;
+import static utils.TestUtils.main;
+
+public class GroupRetryIT
+{
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+
+    @Rule
+    public TemporaryDigdagServer server = TemporaryDigdagServer.of();
+
+    private Path config;
+    private Path projectDir;
+
+    @Before
+    public void setUp()
+            throws Exception
+    {
+        config = folder.newFile().toPath();
+
+        projectDir = folder.getRoot().toPath().resolve("group_retry");
+        Files.createDirectory(projectDir);
+    }
+
+    @Test
+    public void testGroupRetry()
+            throws Exception
+    {
+        DigdagClient client = DigdagClient.builder()
+                .host(server.host())
+                .port(server.port())
+                .build();
+
+        // Push the project
+        pushRevision("acceptance/group_retry/retry-1.dig", "retry");
+
+        // Start the workflow
+        Id originalAttemptId;
+        {
+            CommandStatus startStatus = main("start",
+                    "-c", config.toString(),
+                    "-e", server.endpoint(),
+                    "retry", "retry",
+                    "--session", "now");
+            assertThat(startStatus.errUtf8(), startStatus.code(), is(0));
+            originalAttemptId = getAttemptId(startStatus);
+        }
+
+        // Wait for the attempt to fail
+        assertThat(joinAttempt(client, originalAttemptId).getSuccess(), is(false));
+
+        assertOutputExists(originalAttemptId + "1-1", true);
+        assertOutputExists(originalAttemptId + "1-2a", true);
+        assertOutputExists(originalAttemptId + "1-2b", true);
+        assertOutputExists(originalAttemptId + "1-2c", false);
+
+        // Push a new revision
+        pushRevision("acceptance/group_retry/retry-2.dig", "retry");
```

Comment:
Project name and workflow name and command are all same name **"retry"** ?
It ma be a bit confused for me.

Reply:
I agree with you.. sorry for confusing, let me update project and workflow name.